recipes/marray/findhdf5_fix.diff

```diff
@@ -0,0 +1,26 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f39b35a..a85dbc5 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -44,7 +44,20 @@ endif()
+ # ------------------------------------------------------------------------------
+ # hdf5
+ # ------------------------------------------------------------------------------
+-find_package(HDF5)
++
++# http://t3hmikez0r.com/2014/07/11/cmake-and-hdf5-revisited/
++# back up any user-defined HDF5_DIR setting
++set(_SAVED_HDF5_DIR ${HDF5_DIR})
++# First look for hdf5-config.cmake in defined locations.
++# This file is generated by HDF5 team and is likely more up to date than FindHDF5.cmake included in CMake.
++FIND_PACKAGE(HDF5 NO_MODULE)
```

Comment:
It seems like ``HDF5_INCLUDE_DIR`` is not set any more at all by ``FindHDF5.cmake``.